onnxruntime/python/onnxruntime_cuda_temp_env.py

```diff
@@ -0,0 +1,61 @@
+import os
+import platform
+import re
+import site
+
+
+class TemporaryEnv:
+    def __init__(self, updates):
+        self.original_env = os.environ.copy()
+        os.environ.update(updates)
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        os.environ.clear()
+        os.environ.update(self.original_env)
+
+
+def get_nvidia_dll_paths() -> str:
+    # Get the site-packages path where nvidia packages are installed
+    site_packages_path = site.getsitepackages()[0]
+    nvidia_path = os.path.join(site_packages_path, "nvidia")
+
+    # Collect all directories under site-packages/nvidia that contain .dll files (for Windows)
+    dll_paths = []
+    for root, files in os.walk(nvidia_path):
```

Comment:
for root, dirs, files in os.walk(nvidia_path)
See https://www.tutorialspoint.com/python/os_walk.htm

Reply:
added _, between root and files.